addons/point_of_sale/static/tests/tours/customer_display_tour.js

```diff
@@ -47,5 +64,17 @@ registry.category("web_tour.tours").add("CustomerDisplayTour", {
             },
             Order.doesNotHaveLine({}),
             amountIs("Total", "0.00"),
+            ...Array.from({ length: 2 }).map(() => ({
+                trigger: "body",
+                run: () => postMessage(addRandomProducts(20), "add products").run(),
+            })),
+            {
+                trigger: ".order-container",
+                run: function () {
+                    if (this.anchor.scrollTop === 0) {
```

Comment:
since the auto scroll is 'smooth', it might take some time to scroll all the way to the bottom, so I am just checking that it has started scrolling (i.e. at least has scrolled a bit)